runtime/devnet/src/lib.rs

```diff
@@ -663,6 +663,7 @@ construct_runtime!(
 		Nfts: pallet_nfts = 50,
 		NftFractionalization: pallet_nft_fractionalization = 51,
 		Assets: pallet_assets::<Instance1> = 52,
+		Fungibles: pallet_fungibles = 53,
```

Comment:
Suggest an API section, leaving a decent gap for future SDK pallets.

Reply:
Took 150 with 255 as max. Leaving space for 100 pallets and 100 api pallets